client-js/app/elements/labrad-nodes.ts

```diff
@@ -118,6 +121,27 @@ export class LabradInstanceController extends polymer.Base {
   }
 
 
+  async doAutostart() {
```

Comment:
maybe call this `toggleAutostart`, since it just modifies the list, doesn't actually trigger the anything to be autostarted.